src/main/java/de/tum/in/www1/artemis/service/connectors/athena/AthenaDTOConverterService.java

```diff
@@ -63,7 +60,7 @@ public AthenaDTOConverterService(TextBlockRepository textBlockRepository, TextEx
      * @param exercise the exercise to convert
      * @return *ExerciseDTO for Athena
      */
-    public ExerciseDTO ofExercise(Exercise exercise) {
+    public ExerciseBaseDTO ofExercise(de.tum.in.www1.artemis.domain.Exercise exercise) {
```

Comment:
Since the DTOs were re-named, we should be able to import Exercise and the other entities to avoid using these long names.